tests/do_tests.mingw

```diff
@@ -5,15 +5,8 @@ echo "MinGW test script starting"
 for f in test*.exe
 do
   t=`echo $f | sed s/.exe//`
-  `$f > $t.out`
-  diff -qb "$t.out" "$t.ans"
-  if [ $? -eq 1 ]
-  then
-    echo "$f failed"
-  else
-    true # can't have empty else
-    #echo "$f passed"
-  fi
+  "./$f" > "$t.out"
+  diff -b "$t.out" "$t.ans" > "$t.diff" && rm "$t.diff" || echo "$f failed"
```

Comment:
Is this change needed? Does MinGW Bash not support `$?` or something?

(The one-line change of `$f > $t.out` to `"./$f" > "$t.out"` seems fine. I don't know why the old line originally had backticks around it.)